src/main/models/vehicles/TrainSize.java

```diff
@@ -0,0 +1,25 @@
+package main.models.vehicles;
+
+
+public enum TrainSize{
```

Comment:
Nit: It is not usually recommended to have this much stuff going on in `enum` class

Reply:
Thanks for the note, will keep it in mind for the future.